src/components/chatbox/ai-feature-toast.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect } from 'react'
+import Cookies from 'js-cookie'
+import { IconWand24 } from '@hashicorp/flight-icons/svg-react/wand-24'
+
+import { toast, ToastColor } from 'components/toast'
+import useAuthentication from 'hooks/use-authentication'
+
+import { CmdCtrlIcon } from 'components/command-bar/components/cmd-ctrl-icon'
+import { KIcon } from 'components/command-bar/components/k-icon'
+import Badge from 'components/badge'
+
+const COOKIE_IGNORE_AI_TOAST = 'dev-dot-ignore-ai-toast'
+const COOKIE_HAS_SEEN_AI_TOAST = 'dev-dot-has-seen-ai-toast'
+const COOKIE_EXPIRES_AT = new Date('12/25/2024') // arbitrary date
+
+const TOAST_AUTO_DISMISS_MS = 15000
+
+export function AIFeatureToast() {
+	const { session } = useAuthentication()
+
+	useEffect(() => {
+		// skip toast if we're past the expiration date
+		if (new Date() > COOKIE_EXPIRES_AT) {
+			// return
```

Comment:
❓ Curious why the `return` statements here commented out - are we avoiding having these conditionals active until some future state? (Without the `returns` uncommented, seems like the toast would _always_ pop up if `session?.meta.isAIEnabled` is truthy, even if it's been previously dismissed!)

Reply:
Thanks for catching this! (Accidentally committed testing code)
fixed in 240a273af